client/lib/plans/features-list.js

```diff
@@ -1212,7 +1228,10 @@ export const FEATURES_LIST = {
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Scan Daily (automated)' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Scan (daily, automated)' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.